enterprise/server/test/integration/remote_bazel/remote_bazel_test.go

```diff
@@ -137,6 +139,23 @@ func resetFlags(t *testing.T) {
 	require.NoError(t, err)
 }
 
+// Run remote bazel in a separate process so it doesn't interfere with
+// the local server and cause a race condition.
+func runRemoteBazelInSeparateProcess(t *testing.T, workDir string, serverAddress string, args ...string) {
+	cmd := testcli.Command(t, workDir, append(
+		[]string{
+			"remote",
+			fmt.Sprintf("--remote_runner=%s", serverAddress),
```

Comment:
Tiny nit (also applies to other `Sprintf` calls): `"--remote_runner="+serverAddress` is just as clear.

Reply:
Personal preference :) I prefer Sprintf because I find it easier to modify the strings if I need in the future

Reply:
I also like using string concatenation for flags because it makes all the flags nicely left-aligned in the list, with each list entry starting with `"--flagname=` making it easier to visually scan through the flags and see which ones are being set. If I need to use `fmt.Sprintf()` I will usually do `"--name=" + fmt.Sprintf("%d", someValue)`, formatting the value on its own. (The current code is totally fine / no need to change it, just pointing out a minor readability benefit in case it's not obvious)